src/bitcoin/TxInput.mo

```diff
@@ -60,12 +60,9 @@ module {
       var outputOffset = 0;
 
       let prevTxId = Blob.toArray(prevOutput.txid);
-      let reversedPrevTxid = Array.tabulate<Nat8>(32, func (n : Nat) {
-        prevTxId[prevTxId.size() - 1 - n];
-      });
 
       // Write prevTxId.
-      Common.copy(output, outputOffset, reversedPrevTxid, 0, 32);
+      Common.copy(output, outputOffset, prevTxId, 0, 32);
```

Comment:
As far as I know, the bytes do need to be reversed, so I'm not sure what is happening here.
Are they reversed elsewhere, too, so that we accidentally reversed them twice?